apps/files_sharing/lib/Controller/ShareAPIController.php

```diff
@@ -2025,6 +2060,71 @@ private function checkInheritedAttributes(IShare $share): void {
 				}
 			}
 		}
+	}
+
+	/**
+	 * Send a mail notification again for a share.
+	 * The mail_send option must be enabled for the given share.
+	 * @param string $id the share ID
+	 * @param string $password optional, the password to check against. Necessary for password protected shares.
```

Comment:
```suggestion
	 * @param string $password the password to check against. Necessary for password protected shares.
```

No need to say it's optional, since it has a default value